roles/manage-user-access/README.md

```diff
@@ -0,0 +1,69 @@
+update_access role
+=========
+
+This role updates the password and authorized_key for the provided user
+
+## Requirements
+This role needs to be executed by a user with 'become' privileges
+
+An easy way to create an encrypted password is with the following command:
+
+```
+python -c 'import crypt; print crypt.crypt("password", crypt.mksalt(crypt.METHOD_SHA512))'
+```
+
+## Role Variables
+
+The following variables controls the functionality of `update_access` role
+
+| Name | Purpose|Required|
+|---|---|---|
+|user_name|Identify the username that should be update|Yes|
+|user_name_password|Provide the encrypted password that the username should be set to|No|
+|authorized_keyfile|Provide the path to the keyfile that should be installed for the user|No|
+
+If the `authorized_keyfile` is not set, authorized_keys file update is ignored for the `user_name`
```

Comment:
Can we invert this write-up - i.e.: instead of saying "is not set", say something like ... "If the `authorized_keyfile` is set, the authorized_keys file will be updated for the `user_name`"